ek9000App/src/ekUtil.cpp

```diff
@@ -17,6 +17,8 @@
 
 #include "devEK9000.h"
 
+#include "terminal_types.g.h"
```

Comment:
What's the significance of the `.g` suffix?

Reply:
`.g.` means generated. I ~~copied~~ borrowed this convention from protobuf which uses `.g.h` for generated headers iirc. `.generated.h` or `_generated.h` is also an option here

Reply:
Or maybe no suffix at all, how it was before :P

Reply:
Either way is fine - just was curious